src/gui/sdl_mapper.cpp

```diff
@@ -2271,18 +2274,23 @@ static void MAPPER_SaveBinds(void) {
 	}
 	fclose(savefile);
 	change_action_text("Mapper file saved.",CLR_WHITE);
+	LOG_MSG("MAPPER: Wrote key bindings to %s", filename);
 }
 
-static bool MAPPER_LoadBinds(void) {
-	FILE * loadfile=fopen(mapper.filename.c_str(),"rt");
-	if (!loadfile) return false;
+static bool MAPPER_CreateBindsFromFile(void) {
+	const char* filename = mapper.filename.c_str();
+	FILE * loadfile = fopen(filename,"rt");
+	if (!loadfile)
+		return false;
 	ClearAllBinds();
+	uint32_t bind_tally = 0;
 	char linein[512];
 	while (fgets(linein,512,loadfile)) {
 		CreateStringBind(linein);
+		++bind_tally;
 	}
 	fclose(loadfile);
-	LOG_MSG("MAPPER: Loading mapper settings from %s", mapper.filename.c_str());
+	LOG_MSG("MAPPER: Loaded %d key bindings from %s", bind_tally, filename);
```

Comment:
This is a very nice touch - this additional log makes big differnce during testing :)